entrypoint.sh

```diff
@@ -41,12 +41,26 @@ cloud-api-adaptor-libvirt libvirt \
 	-socket /run/peerpod/hypervisor.sock
 }
 
+vsphere() {
+set -x
+cloud-api-adaptor-vsphere vsphere \
+        -vcenter-url ${GOVC_URL}  \
+	-user-name ${GOVC_USERNAME} \
+	-password ${GOVC_PASSWORD} \
+	-data-store datastore2 \
+	-pods-dir /run/peerpod/pods \
+	-deploy-folder peerpods \
+	-template podvm-new-template \
```

Comment:
should template be configurable? any idea what is data store?

Reply:
`template` is equivalent to ec2 launch template. It is configurable. For this iteration let's go with default and we'll change it later on.